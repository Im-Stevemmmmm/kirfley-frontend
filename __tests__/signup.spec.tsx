import { MockedProvider } from "@apollo/client/testing";
import { render } from "@testing-library/react";
import Signup from "pages/signup";

describe("index page", () => {
    it("renders correctly", () => {
        const { container } = render(
            <MockedProvider>
                <Signup />
            </MockedProvider>
        );

        expect(container).toMatchSnapshot();
    });
});
