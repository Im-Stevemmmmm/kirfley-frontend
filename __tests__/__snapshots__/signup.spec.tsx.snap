// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index page renders correctly 1`] = `
<div>
  <div
    id="container"
  >
    <title>
      Sign up | 
      Kirfley
    </title>
    <main
      id="form"
    >
      <form
        class="form"
      >
        <h1
          class="form__title"
        >
          Sign Up
        </h1>
        <label
          class="form__label"
          for="username"
        >
          Username
        </label>
        <input
          class="form__input"
          id="username"
          name="username"
          placeholder="Atleast 4 characters long"
          type="text"
          value=""
        />
        <label
          class="form__label"
          for="email"
        >
          Email
        </label>
        <input
          class="form__input"
          id="email"
          name="email"
          placeholder="example@email.com"
          type="text"
          value=""
        />
        <label
          class="form__label"
          for="password"
        >
          Password
        </label>
        <input
          class="form__input"
          id="password"
          name="password"
          placeholder="Atleast 4 characters long"
          type="password"
          value=""
        />
        <label
          class="form__label"
          for="confirmPassword"
        >
          Confirm Password
        </label>
        <input
          class="form__input"
          id="confirmPassword"
          name="confirmPassword"
          type="password"
          value=""
        />
        <div
          class="buttonContainer"
        >
          <button
            class="form__button inline"
            type="submit"
          >
            Sign Up
          </button>
          <p
            class="buttonContainer__text"
          >
            or
          </p>
          <button
            class="form__button inline"
            type="button"
          >
            Back to log in
          </button>
        </div>
      </form>
    </main>
  </div>
</div>
`;
